WcaOnRails/config/locales/en.yml

```diff
@@ -269,10 +269,10 @@ en:
         refund_policy_percent: "Percent to be refunded"
         refund_policy_limit_date: "Limit date for refunds"
         guests_entry_fee_lowest_denomination: "Entry fee for guests"
-        regulation_z1: "I would like to require competitors to submit puzzles during registration as per Regulation Z1)"
-        regulation_z1_reason: "Reason for Z1)"
-        regulation_z3: "I would like to select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions as per Regulation Z3)"
-        regulation_z3_reason: "Reason for Z3)"
+        regulation_z1: "I would like to require competitors to submit puzzles at a specific time before an event is scheduled to begin"
+        regulation_z1_reason: "Reason for submitting puzzles early"
+        regulation_z3: "I would like to select competitors who directly qualify for events at this competition based on the results of specific previous competitions"
```

Comment:
I would re-write this sentence, as it doesn't sound quite good. Also, maybe we should include another field for the optional requirement of forbidding certain combinations of events. Or maybe this last part can go together with #4217.

Reply:
Sure or be its own thing.
I tried to stay away from modifying the form. I only did it for the 28 day thing since that was easy enough and causes WCAT issues, whereas we usually catch the combination of events, so that addition isn't as urgent. I've added it to the WCAT wish list: https://github.com/thewca/worldcubeassociation.org/issues/3759#issuecomment-515283243

Reply:
I adjusted the sentence as well as changed the reasons (to include a description of what is being requested)

Reply:
Perfect!